test/lib/PhaserTester.ts

```diff
@@ -0,0 +1,116 @@
+import 'phaser';
+import { Scene, Game, GameObjects } from 'phaser';
+
+type SceneNewable<T extends Scene> = {
+    new (...args: any[]): T;
+};
+
+abstract class TestScene extends Scene {
+    public abstract onTestStart(testCallback): void;
+}
+
+export default class PhaserTester<T extends TestScene> {
+    private game: Game;
+    private scene: T;
+    private sceneNewable: SceneNewable<T>;
+    private testText: GameObjects.Text;
+
+    public keyDown: EventListenerOrEventListenerObject;
+    public keyUp: EventListenerOrEventListenerObject;
+
+    constructor(sceneNewable: SceneNewable<T>) {
+        this.sceneNewable = sceneNewable;
+    }
+
+    public getScene(): T {
+        return this.scene;
+    }
+
+    public getGameWidth() {
+        return this.game.canvas.width;
```

Comment:
This method is currently only used in one of the moving platform tests, and it would work if it weren't for the fact that moving platforms check for hardcoded bounds. To be addressed in #46